paddle/fluid/framework/ir/node.h

```diff
@@ -92,6 +93,7 @@ class Node {
   Node() = delete;
 
   static int count_;
+  // Please don't use this API or make this public.
```

Comment:
How about add some guide document for how to develop an ir pass, when I adding a pass that need to copy all nodes and reconstruct the graph, I have to copy code from other passes.

Reply:
Sure